2021-12-17 00:45:08 [info]: API listening at http://localhost:9000
2021-12-17 00:45:52 [info]: ::1 (POST) /users
2021-12-17 00:45:52 [error]: Error: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Handshake.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Handshake.js:123:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Protocol._enqueue (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:48:16)
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.findUsersByHandle (C:\Users\larth\Coding\streamfinder\api\controllers\user.js:41:21)
    at User.insert (C:\Users\larth\Coding\streamfinder\api\models\user.js:97:44)
2021-12-17 00:48:42 [info]: ::1 (POST) /users
2021-12-17 00:48:42 [error]: Error: ER_BAD_DB_ERROR: Unknown database 'streamfinder'
    at Handshake.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Handshake.js:123:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Protocol._enqueue (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:48:16)
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:202:8)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.findUsersByHandle (C:\Users\larth\Coding\streamfinder\api\controllers\user.js:41:21)
    at User.insert (C:\Users\larth\Coding\streamfinder\api\models\user.js:97:44)
2021-12-17 00:48:52 [info]: API listening at http://localhost:9000
2021-12-17 00:48:54 [info]: ::1 (POST) /users
2021-12-17 00:48:54 [error]: Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertUser (C:\Users\larth\Coding\streamfinder\api\controllers\user.js:31:21)
    at User.insert (C:\Users\larth\Coding\streamfinder\api\models\user.js:104:30)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:49:17 [info]: ::1 (POST) /users
2021-12-17 00:49:17 [error]: Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertUser (C:\Users\larth\Coding\streamfinder\api\controllers\user.js:31:21)
    at User.insert (C:\Users\larth\Coding\streamfinder\api\models\user.js:104:30)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:49:21 [info]: API listening at http://localhost:9000
2021-12-17 00:49:22 [info]: ::1 (POST) /users
2021-12-17 00:49:22 [error]: Error: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect integer value: 'ef92b778bafe771e89245b89ecbc08a44a4e166c06659911881f383d4473e94f' for column 'email_verified' at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertUser (C:\Users\larth\Coding\streamfinder\api\controllers\user.js:31:21)
    at User.insert (C:\Users\larth\Coding\streamfinder\api\models\user.js:104:30)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:50:22 [info]: API listening at http://localhost:9000
2021-12-17 00:50:23 [info]: ::1 (POST) /users
2021-12-17 00:51:54 [info]: ::1 (GET) /auth
2021-12-17 00:51:54 [error]: Error: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'created' at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertToken (C:\Users\larth\Coding\streamfinder\api\controllers\token.js:22:21)
    at C:\Users\larth\Coding\streamfinder\api\models\token.js:54:47
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:55:00 [info]: ::1 (GET) /auth
2021-12-17 00:55:00 [error]: Error: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'created' at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertToken (C:\Users\larth\Coding\streamfinder\api\controllers\token.js:22:21)
    at C:\Users\larth\Coding\streamfinder\api\models\token.js:54:47
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:55:06 [info]: API listening at http://localhost:9000
2021-12-17 00:55:07 [info]: ::1 (GET) /auth
2021-12-17 00:55:07 [error]: Error: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'created' at row 1
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertToken (C:\Users\larth\Coding\streamfinder\api\controllers\token.js:22:21)
    at C:\Users\larth\Coding\streamfinder\api\models\token.js:54:47
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 00:56:19 [info]: ::1 (GET) /auth
2021-12-17 00:59:50 [info]: ::1 (GET) /users
2021-12-17 01:00:11 [info]: ::1 (GET) /users
2021-12-17 01:00:38 [info]: ::1 (POST) /titles
2021-12-17 01:00:38 [error]: Error: ER_NO_SUCH_TABLE: Table 'streamdex.titles' doesn't exist
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.getTitlesByID (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:7:21)
    at getNewID (C:\Users\larth\Coding\streamfinder\api\models\title.js:160:28)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\title.js:112:26)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\show.js:33:21)
    at C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:31:26
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 01:02:27 [info]: ::1 (POST) /titles
2021-12-17 01:02:27 [error]: Error: ER_NO_SUCH_TABLE: Table 'streamdex.titles' doesn't exist
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.getTitlesByID (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:7:21)
    at getNewID (C:\Users\larth\Coding\streamfinder\api\models\title.js:160:28)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\title.js:112:26)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\show.js:33:21)
    at C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:31:26
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 01:02:32 [info]: API listening at http://localhost:9000
2021-12-17 01:02:33 [info]: ::1 (POST) /titles
2021-12-17 01:02:33 [error]: Error: ER_NO_SUCH_TABLE: Table 'streamdex.titles' doesn't exist
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.getTitlesByID (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:7:21)
    at getNewID (C:\Users\larth\Coding\streamfinder\api\models\title.js:160:28)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\title.js:112:26)
    at Show.insert (C:\Users\larth\Coding\streamfinder\api\models\show.js:33:21)
    at C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:31:26
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 01:03:19 [info]: ::1 (POST) /titles
2021-12-17 01:03:24 [info]: ::1 (GET) /titles
2021-12-17 01:04:28 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:04:49 [info]: ::1 (POST) /titles/6e2549079cfe83ddd0f882486230dd5f/seasons
2021-12-17 01:04:49 [error]: Error: ER_BAD_FIELD_ERROR: Unknown column 'trailer' in 'field list'
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertSeason (C:\Users\larth\Coding\streamfinder\api\controllers\season.js:22:21)
    at Season.insert (C:\Users\larth\Coding\streamfinder\api\models\season.js:77:32)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 01:05:21 [info]: ::1 (POST) /titles/6e2549079cfe83ddd0f882486230dd5f/seasons
2021-12-17 01:05:37 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f/seasons
2021-12-17 01:05:37 [error]: Error: ER_BAD_FIELD_ERROR: Unknown column 'timestamp' in 'field list'
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.getSeasonLinks (C:\Users\larth\Coding\streamfinder\api\controllers\season.links.js:6:21)
    at Season.init (C:\Users\larth\Coding\streamfinder\api\models\season.js:69:71)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Promise.all (index 0)
2021-12-17 01:06:35 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f/seasons
2021-12-17 01:07:25 [info]: ::1 (POST) /seasons/e743193a51ccad01d6b112298a50bd7f/episodes
2021-12-17 01:07:36 [info]: ::1 (GET) /seasons/e743193a51ccad01d6b112298a50bd7f/episodes
2021-12-17 01:12:03 [info]: ::1 (GET) /pages/new
2021-12-17 01:12:15 [info]: ::1 (GET) /titles/?q=test
2021-12-17 01:12:15 [error]: Error: ER_FT_MATCHING_KEY_NOT_FOUND: Can't find FULLTEXT index matching the column list
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.searchAllTitles (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:15:21)
    at Object.getTitles (C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:157:38)
    at C:\Users\larth\Coding\streamfinder\api\routes\titles.js:11:18
    at Layer.handle [as handle_request] (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:112:3)
2021-12-17 01:14:44 [info]: ::1 (GET) /titles/?q=test
2021-12-17 01:14:44 [error]: Error: ER_FT_MATCHING_KEY_NOT_FOUND: Can't find FULLTEXT index matching the column list
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.searchAllTitles (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:15:21)
    at Object.getTitles (C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:157:38)
    at C:\Users\larth\Coding\streamfinder\api\routes\titles.js:11:18
    at Layer.handle [as handle_request] (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:112:3)
2021-12-17 01:15:24 [info]: ::1 (GET) /titles/?q=test
2021-12-17 01:15:24 [error]: Error: ER_FT_MATCHING_KEY_NOT_FOUND: Can't find FULLTEXT index matching the column list
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.searchAllTitles (C:\Users\larth\Coding\streamfinder\api\controllers\title.js:15:21)
    at Object.getTitles (C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:157:38)
    at C:\Users\larth\Coding\streamfinder\api\routes\titles.js:11:18
    at Layer.handle [as handle_request] (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\larth\Coding\streamfinder\api\node_modules\express\lib\router\route.js:112:3)
2021-12-17 01:17:31 [info]: ::1 (GET) /titles/?q=test
2021-12-17 01:17:31 [info]: ::1 (GET) /images/
2021-12-17 01:17:37 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:17:37 [info]: ::1 (GET) /images/
2021-12-17 01:20:23 [info]: ::1 (GET) /auth
2021-12-17 01:20:23 [info]: ::1 (GET) /users/self
2021-12-17 01:20:29 [info]: ::1 (GET) /users
2021-12-17 01:20:29 [info]: ::1 (GET) /users/self
2021-12-17 01:20:31 [info]: ::1 (GET) /users/28eb5ad4bdc905f3
2021-12-17 01:20:35 [info]: ::1 (GET) /images/
2021-12-17 01:20:35 [info]: ::1 (GET) /titles
2021-12-17 01:20:35 [info]: ::1 (GET) /users/self
2021-12-17 01:20:35 [info]: ::1 (GET) /images/
2021-12-17 01:20:37 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:20:37 [info]: ::1 (GET) /images/
2021-12-17 01:20:43 [info]: ::1 (GET) /images
2021-12-17 01:20:43 [info]: ::1 (GET) /users/self
2021-12-17 01:20:56 [info]: ::1 (POST) /images/
2021-12-17 01:20:56 [info]: ::1 (GET) /images
2021-12-17 01:20:56 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8/info
2021-12-17 01:20:56 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:20:56 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:21:09 [info]: ::1 (PUT) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:21:14 [info]: ::1 (GET) /images/
2021-12-17 01:21:14 [info]: ::1 (GET) /titles
2021-12-17 01:21:14 [info]: ::1 (GET) /images/
2021-12-17 01:21:14 [info]: ::1 (GET) /users/self
2021-12-17 01:21:15 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:21:15 [info]: ::1 (GET) /images/
2021-12-17 01:21:17 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:21:18 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:21:18 [info]: ::1 (GET) /images/
2021-12-17 01:21:21 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:21:21 [info]: ::1 (GET) /images/
2021-12-17 01:21:38 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:21:38 [info]: ::1 (GET) /images/
2021-12-17 01:23:12 [info]: ::1 (GET) /images/
2021-12-17 01:23:12 [info]: ::1 (GET) /images/
2021-12-17 01:23:12 [info]: ::1 (GET) /titles
2021-12-17 01:23:12 [info]: ::1 (GET) /users/self
2021-12-17 01:23:13 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:23:13 [info]: ::1 (GET) /images/
2021-12-17 01:23:15 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:23:17 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:23:17 [info]: ::1 (GET) /images/
2021-12-17 01:23:27 [info]: ::1 (GET) /images/
2021-12-17 01:23:27 [info]: ::1 (GET) /images/
2021-12-17 01:23:27 [info]: ::1 (GET) /titles
2021-12-17 01:23:27 [info]: ::1 (GET) /users/self
2021-12-17 01:23:28 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:23:28 [info]: ::1 (GET) /images/
2021-12-17 01:23:30 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:23:31 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:23:31 [info]: ::1 (GET) /images/
2021-12-17 01:58:36 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:58:36 [info]: ::1 (GET) /images/
2021-12-17 01:58:39 [info]: API listening at http://localhost:9000
2021-12-17 01:58:44 [info]: ::1 (GET) /images/
2021-12-17 01:58:44 [info]: ::1 (GET) /titles
2021-12-17 01:58:44 [info]: ::1 (GET) /users/self
2021-12-17 01:58:44 [info]: ::1 (GET) /images/
2021-12-17 01:58:45 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:58:45 [info]: ::1 (GET) /images/
2021-12-17 01:58:47 [info]: ::1 (GET) /images/(val)%20=%3E%20val%20===%20null%20%7C%7C%20(val%20!==%20null%20&&%20typeof%20val%20===%20%22number%22)
2021-12-17 01:58:48 [info]: ::1 (GET) /images/
2021-12-17 01:58:51 [info]: ::1 (GET) /images
2021-12-17 01:58:51 [info]: ::1 (GET) /users/self
2021-12-17 01:58:51 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:58:51 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8/info
2021-12-17 01:58:51 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:58:54 [info]: ::1 (GET) /images/
2021-12-17 01:58:54 [info]: ::1 (GET) /images/
2021-12-17 01:58:54 [info]: ::1 (GET) /titles
2021-12-17 01:58:54 [info]: ::1 (GET) /users/self
2021-12-17 01:58:55 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:58:55 [info]: ::1 (GET) /images/
2021-12-17 01:58:56 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:58:58 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:58:58 [info]: ::1 (GET) /images/
2021-12-17 01:59:41 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:59:41 [info]: ::1 (GET) /images/
2021-12-17 01:59:46 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:59:47 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:59:47 [info]: ::1 (GET) /images/
2021-12-17 01:59:47 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:59:47 [info]: ::1 (GET) /images/
2021-12-17 01:59:47 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 01:59:53 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 01:59:54 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 02:00:26 [info]: ::1 (POST) /titles/6e2549079cfe83ddd0f882486230dd5f/links/netflix
2021-12-17 02:00:26 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:00:26 [error]: Error: ER_NO_REFERENCED_ROW: Cannot add or update a child row: a foreign key constraint fails
    at Query.Sequence._packetToError (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:290:12)
    --------------------
    at Pool.query (C:\Users\larth\Coding\streamfinder\api\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\larth\Coding\streamfinder\api\util.js:41:18
    at new Promise (<anonymous>)
    at Object.dbPromise (C:\Users\larth\Coding\streamfinder\api\util.js:40:16)
    at Object.insertLink (C:\Users\larth\Coding\streamfinder\api\controllers\links.js:14:21)
    at Link.insert (C:\Users\larth\Coding\streamfinder\api\models\links.js:238:31)
    at C:\Users\larth\Coding\streamfinder\api\model_functions\title.js:208:25
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
2021-12-17 02:03:49 [info]: ::1 (POST) /titles/6e2549079cfe83ddd0f882486230dd5f/links/netflix
2021-12-17 02:03:49 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:03:49 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:03:49 [info]: ::1 (GET) /images/
2021-12-17 02:03:49 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 02:03:53 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:03:54 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 02:04:14 [info]: ::1 (POST) /seasons/e743193a51ccad01d6b112298a50bd7f/availability/netflix
2021-12-17 02:04:14 [info]: ::1 (PUT) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:04:14 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:04:14 [info]: ::1 (GET) /images/
2021-12-17 02:04:14 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
2021-12-17 02:04:16 [info]: ::1 (GET) /titles/6e2549079cfe83ddd0f882486230dd5f
2021-12-17 02:04:17 [info]: ::1 (GET) /images/9c150de3298955c528a28aaf24da3dd8
